data/data-pipeline/data_pipeline/etl/sources/tribal_overlap/etl.py

```diff
@@ -72,16 +74,17 @@ def _create_string_from_list(series: pd.Series) -> str:
         str_list = sorted(str_list)
         return ", ".join(str_list)
 
-    @staticmethod
+    @classmethod
     def _adjust_percentage_for_frontend(
+        cls,
         percentage_float: float,
     ) -> Optional[float]:
         """Round numbers very close to 0 to 0 and very close to 1 to 1 for display"""
         if percentage_float is None:
             return None
         if percentage_float < 0.01:
```

Comment:
should we have the same cutoff type here? like 0.005 in the vein of 0.995? There's a little bit of asymmetry?

Reply:
The decisions were made at different times, so good catch. @BethMattern thoughts on changing the cutoff for "very close to 0%" smaller (.5% instead of 1%)?

Reply:
If we change this, it'll have both a backend and frontend change, and it's not obvious to me we should, so I've pulled it into its own ticket to explicitly make the decision: #2006